include/xtensor/xreducer.hpp

```diff
@@ -608,6 +608,14 @@ namespace xt
         {
             return make_xreducer_functor(get_reduce(), get_init().template rebind<NT>(), get_merge());
         }
+
+    private:
+        
+        // Workaround for clang-cl
+        const base_type& upcast() const
+        {
+            return static_cast<const base_type&>(*this);
+        }
```

Comment:
Eventually, we may want to add this to the list of compiler workarounds.

Reply:
Yes I was just testing if this was the cause for the failure.

Reply:
I will do that, thanks for the review